src/gatsby/src/styles/pageContent/pageMenu.scss

```diff
@@ -80,7 +80,7 @@
       width: 100%;
       height: 68px !important;
       border: none;
-      padding: 0px 20px;
+      padding: 0 32px 0 20px;
```

Comment:
This works, but I noticed that the edge of the icon is right up against the edge of the text cutoff with this padding:
<img width="172" alt="image" src="https://github.com/user-attachments/assets/2efdf643-8c30-4b2a-99d0-ca4dfa71a9af" />
You might consider increasing it to 40px or so, to add a bit of space:

<img width="135" alt="image" src="https://github.com/user-attachments/assets/550cd040-c1c4-447a-a471-2bc8879606d4" />